llm_evaluation/run.py

```diff
@@ -548,8 +562,27 @@ def evaluate_task(seq_idx: int, prediction: Dict[str, Any]) -> bool:
     )
     logger.info("=" * 60)
 
+    # Load robustness predictions if requested
+    robustness_predictions = None
+    if robustness_predictions_path:
+        try:
+            robustness_predictions = load_predictions_from_path(
+                robustness_predictions_path
+            )
+            logger.info(
+                f"Loaded robustness predictions from {robustness_predictions_path}"
+            )
+        except FileNotFoundError:
+            logger.warning(
+                f"Robustness predictions not found at {robustness_predictions_path}"
+            )
+        except Exception as e:
+            logger.warning(
+                f"Could not load robustness predictions from {robustness_predictions_path}: {e}"
+            )
+
```

Comment:
Is the code designed to continue running after capture the error? Why not just raise Exception or Error?